library/src/engine.cxx

```diff
@@ -270,6 +270,12 @@ void engine::loadPlugin(const std::string& pathOrName, const std::vector<fs::pat
     return;
   }
 
+  // For easier removal when removing deprecation: F3D_DEPRECATED
+  if (pathOrName == "exodus") {
+    f3d::log::warn("The 'exodus' plugin is deprecated, load 'hdf' instead");
+    return f3d::engine::loadPlugin("hdf");
```

Comment:
it may be better to copy pathOrName into a non const local variable and edit it here so that the whole logic can work with `hdf` instead of force loading it ?

Reply:
I thought using a recursive call was cleaner than using another temporary variable, but as you wish. I also notice that the call is missing the second parameter

Reply:
ha, its a recursive call! I missed that. Current impl is fine.